nilearn/plotting/html_document.py

```diff
@@ -164,10 +165,10 @@ def open_in_browser(self, file_name=None, temp_file_lifetime=30):
         if temp_file_lifetime is None:
             if not named_file:
                 warnings.warn(
-                    ("Saved HTML in temporary file: {}\n"
+                    (f"Saved HTML in temporary file: {file_name}\n"
                      "file size is {:.1f}M, delete it when you're done, "
                      "for example by calling this.remove_temp_file").format(
-                         file_name, file_size))
+                         file_size))
```

Comment:
Cna(t you integrate this format into the string ?

Reply:
Yup there is a metric ton of extra fixes to do in this part of the code base.
I added a to-do in the top and left this as a draft because it felt like it was going to take a bit longer than expected.
Will try to not leave the PR grow beyond a 100 line changes. I think I have put all of you through enough of those.